lib/main_common.pm

```diff
@@ -1527,15 +1529,48 @@ sub load_extra_tests_textmode {
     # dstat is not in sle12sp1
     loadtest "console/dstat" if is_sle('12-SP2+') || is_opensuse;
     # MyODBC-unixODBC not available on < SP2 and sle 15 and only in SDK
-    if (sle_version_at_least('12-SP2') && !(sle_version_at_least('15'))) {
+    if (is_sle('12-SP2+') && !(is_sle('15+'))) {
         loadtest "console/mysql_odbc" if check_var_array('ADDONS', 'sdk') || check_var_array('SCC_ADDONS', 'sdk');
     }
     # bind need source package and legacy and development module on SLE15+
-    loadtest 'console/bind' if get_var('MAINT_TEST_REPO');
-    if (is_sle('>=15')) {
-        loadtest 'console/systemd_testsuite';
-    }
+    loadtest 'console/bind'              if get_var('MAINT_TEST_REPO');
+    loadtest 'console/systemd_testsuite' if is_sle('15+');
     loadtest 'console/mdadm' unless is_jeos;
+}
+
+sub load_extra_tests_textmode_other {
```

Comment:
You know, "other" just sounds like another word for "extra" ;) I guess we should get rid of "extra" as a word and just not mention it at all.

Reply:
I know. This time, it is more like "the rest of the tests". I have zypper-test and kdump-test. I even have the console-test as a result of refactoring (those are the common lines in the "old" code and the new code without zypper and kdump). I am short of good names :)

Reply:
As I suggested, what about just getting rid of "extra"? E.g. call it "load_textmode_other"?